nautobot_golden_config/jobs.py

```diff
@@ -94,8 +94,14 @@ class Meta:
     @commit_check
     def run(self, data, commit):  # pylint: disable=too-many-branches
         """Run config compliance report script."""
-        # pylint: disable-msg=too-many-locals
         # pylint: disable=unused-argument
+
+        _ = [
```

Comment:
I've seen this pattern before with .items() where potentially k or v isn't really needed.  Curious if this is needed?  Is this more pythonic to assign it to the unused variable (_) vs just have the code execute the list comprehsion?

Reply:
I tested the 3 jobs with this (using 2 backup/intended repos and using 1 of each), and the jobs work as expected. They did before too so this might just not be needed.

Reply:
It's not a change in expectations if we don't do this. Meaning, before the change the repo's would sync, now they don't. This ensures all repos sync before it runs. which was the expectation prior.

Reply:
I don't think this would necessarily cause a job to fail in a "sunny day" scenario, but if someone went and modified data in the repository itself and this sync didn't happen I'd expect the job to then fail.  I maybe way off on that analysis, just my initial thought.

Reply:
Looks like Ken is right here. I updated the backup file for a device in such a way that it would change the compliance status once compliance was run. Without the `git_wrapper` stuff here the compliance job did not pick up the change. With the `git_wrapper` stuff it did.

Reply:
The list comprehension looks unnecessary to me
I think it would be cleaner like this
```python
for repo in GoldenConfigSetting.objects.first().backup_repository.all():
    git_wrapper(self, repo, "backup")
```

Reply:
> It's a change in expectations if we don't do this. Meaning, before the change the repo's would sync, now they don't. This ensures all repos sync before it runs. which was the expectation prior.

@itdependsnetworks Are you saying that prior to the multi repos PR, the git repositories would be synced prior to executing the compliance job ?

Reply:
@dgarros this was in the place of the lines being added here, prior to the multiple repo support PR getting merged in:
```
        backup_repo = git_wrapper(self, GoldenConfigSetting.objects.first().backup_repository, "backup")
        intended_repo = git_wrapper(self, GoldenConfigSetting.objects.first().intended_repository, "intended")
```

Reply:
@dgarros yes, luckily we have not release, so this will be added before released.